### DISCUSSION TITLE
SimpleLogin premium emails considered as disposable and blocked by Turso

### DISCUSSION THREAD

**Discussion by x80486** [simple-login/app#2112] [Q&A]
I wanted to bring more attention to an issue that many of us keep encountering — numerous websites are blocking SimpleLogin domains, incorrectly labeling them as _disposable email_ services. This creates unnecessary hurdles for users who rely on SimpleLogin for their email management needs.

One example of this is [Turso](https://turso.tech) (to mention just one), which currently blocks all SimpleLogin domains.

![___Screenshot from 2024-05-18 19-09-23](https://github.com/simple-login/app/assets/34633617/54afd215-03fb-4f3e-90cf-db79216d0f82)

I think it would be great if this issue could be addressed proactively.

One approach could be reaching out to the owners of these libraries that aggregate genuinely disposable domains to clarify the nature of SimpleLogin domains and request reconsideration. If left unaddressed, this trend of domain blocking could become more widespread, severely limiting the usability of SimpleLogin across many websites and applications.

**Comment by nguyenkims** [comment]
Thanks for letting us know, can you please contact the websites and send the blocking websites to antiabuse@proton.me ? The team will then also reach out to the websites to ask them to remove SL domains from their blocking list.

More info on https://simplelogin.io/docs/report-blocking-website/

**Comment by trikotret** [comment]
I was able to sign-up with my custom domain that I have with SL.
